app/views/reports/_call_line.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="report-section">
+  <h2 class="border-bottom">Week of <%= week_range %></h2>
+
+  <h3><%= link_to "DC Call Line", '#', class: 'line-header' %></h3>
```

Comment:
Looks like this code repeats a few times, and I bet we can clean it up. I have a few ideas, what do you think is the best one?

* We can either send data to the report in an array as follows and loop thru it:

```
[ 'DC' => { patients_contacted: 46, new_patients_contacted: 23, pledges_sent: 34 }, 'MD' => { ... }, 'VA' => { ... } ]
``` 
 and then do something like:

```rb
<% lines.each do |line| %>
Patients contacted: <%= @data[line][:patients_contacted] %>
...
<% end %>
```

* Or, we could move these to their own partials, and do something like this:

```rb 
# in this partial
<%= render 'reports/contact_report, data: @data['DC'] %>
<%= render 'reports/contact_report, data: @data['MD'] %>
... etc

# in its own subpartial, L4-16 would move
<h3><%= link_to "#{data[:line]} Call Line", '#', class: 'line-header' %></h3>
... etc
```

@ashlynnpai curious as to your thoughts! I think I favor the loop, given what the index looks like. Do you think one sounds more flexible and easier to maintain / expand than the other?